src/restaurant_inspections_api/validations.clj

```diff
@@ -49,3 +51,16 @@
   [validated-map]
   {:invalid (into {} (filter #(false? (second %)) validated-map))
    :valid (into {} (filter #(not (false? (second %)))) validated-map) })
+
+(defn processable?
```

Comment:
I'm not sure we want processable? under validations. But no big deal. I'm trying some things on my own as well to clean things up.